splice_machine/src/main/java/com/splicemachine/derby/impl/storage/SpliceRegionAdmin.java

```diff
@@ -775,14 +778,21 @@ public static  ConglomerateDescriptor getIndex(TableDescriptor td, String indexN
     private static byte[] getRowKey(TableDescriptor td, ConglomerateDescriptor index, ExecRow execRow,
                              String splitKey, String columnDelimiter, String characterDelimiter,
                              String timeFormat, String dateFormat, String timestampFormat) throws Exception{
+        EmbedConnection defaultConn=(EmbedConnection) SpliceAdmin.getDefaultConn();
+        Activation lastActivation=defaultConn.getLanguageConnection().getLastActivation();
+        LanguageConnectionContext lcc = lastActivation.getLanguageConnectionContext();
+
         // set up csv reader
         CsvPreference preference = createCsvPreference(columnDelimiter, characterDelimiter);
         Reader reader = new StringReader(splitKey);
         List<Integer> valueSizeHints = new ArrayList<>(execRow.nColumns());
         for(DataValueDescriptor dvd : execRow.getRowArray()) {
             valueSizeHints.add(dvd.estimateMemoryUsage());
         }
-        MutableCSVTokenizer tokenizer = new MutableCSVTokenizer(reader,preference, false,
+
+        boolean quotedEmptyIsNull = !PropertyUtil.getCachedDatabaseBoolean(
```

Comment:
I think our current behaviour is wrong (treating quoted empty strings as null). Shouldn't our default behaviour be the "DB2-compatible" one?

Reply:
@arnaud-splice brought me up to speed regarding yesterday's discussion.